src/main/java/org/opensearch/timeseries/indices/IndexManagement.java

```diff
@@ -1008,6 +1016,53 @@ public <T> void initCustomResultIndexAndExecute(String resultIndexOrAlias, Execu
         }
     }
 
+    /**
```

Comment:
Would you roll over flattened result index same as custom result index?

Reply:
added flatten result index alias to the rollover and delete flow